notebooks/coolest_template_generator.ipynb

```diff
@@ -0,0 +1,132 @@
+{
```

Comment:
<p>do we need this notebook? It basically demonstrates COOLEST parts, and not SLSim aspects. The current notebook has not a single SLSim line of code used.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LSST-strong-lensing/slsim/pull/127/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Deleted for now. I can add it later with slsim example (updating coolest and creating slsim).